src/overlords/defense/rangedDefense.ts

```diff
@@ -34,8 +36,12 @@ export class RangedDefenseOverlord extends CombatOverlord {
 
 	private handleDefender(hydralisk: CombatZerg): void {
 		if (this.room.hostiles.length > 0) {
+			hydralisk.heal(hydralisk); // heal self if no other targets
```

Comment:
I think this was because hydralisks were healing others at range rather than healing themselves when injured.  So you'd have a group of hydras dying because they were weak healing each other. Will remove but should address.